x-pack/plugins/apm/public/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ApmPlugin } from './plugin';
```

Comment:
I prefer to see the file and content structure of the new plugin created using the `scripts/generate_plugin`